recipe/openid/index.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
```

Comment:
This is not necessarily an issue, but why do we need to expose the openid recipe now?

Reply:
Initially I assume it was needed but it is not. I have removed it.